etc/build/Makefile

```diff
@@ -12,7 +12,13 @@ $(shell mkdir -p $(tmplocation))
 
 bblfsh-sdk := $(shell command -v bblfsh-sdk 2> /dev/null)
 bblfsh-tools := $(shell command -v bblfsh-tools 2> /dev/null)
-ifdef bblfsh-tools
+in-container := $(shell echo $$ENVIRONMENT)
+
+# TODO(bzz) ideal: `go run` ONLY inside container, started from non-linux host
```

Comment:
```
$ file ~/dev/go/bin/bblfsh-tools 
/home/smola/dev/go/bin/bblfsh-tools: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, not stripped
```

Maybe you can check if it's non-linux host if `/go/bin/bblfsh-tools` is not an ELF 64-bit binary?